ADF diagnostic configuration added

s-vishnjakov opened the pull request.

s-vishnjakov committed:
ADF diagnostic configuration added

s-vishnjakov committed:
Update Overview.md

s-vishnjakov commented:
Commit 29d7f69

s-vishnjakov committed:
Delete Overview.md

Commit:
Revert "Delete Overview.md"

This reverts commit 369e75d572922989d78e7d979fecaec44981c9a4.

s-vishnjakov reviewed (commented):
It's the proper commit

Commit:
Screenshots added

reaver89 commented on Overview.md:
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
+Configure diagnostic setting to send platform logs and metrics to Log Analytics Workspace.
+
+![Diagnostic Settings](Artifacts/DiagnosticSettings.PNG)
+1. Go to the menu of the created Data Factory.
```
1. Not "created" but "Data Factory whic you are going to monitor".

s-vishnjakov replied:
edited

reaver89 commented on Overview.md:
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
+3. Click on "+Add diagnostic Setting".
+4. Specify a name for the Diagnostic setting. 
+5. Select the required categories to send: ActivityRuns, PipelineRuns, TriggerRuns, AllMetrics.
+6. Select the subscription that contains VIAcode-Azure-Data-Factory monitoring and workspace itself.
```
I don't see info about  "Destination Details" , which destination table I should select and so on

reaver89 commented on Overview.md:
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
 < screenshot here >  
 Provide this identity with a Reader role to your subscription ([more info](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/howto-assign-access-portal))
+
+## Configure ADF Diagnostic Settings
```
This section is like  "prerequisite", withou Diagnostic settings configuration Workbooks will not work. It's better to put it into the beginning of the guide

s-vishnjakov replied:
Got it.

s-vishnjakov committed:
Update Overview.md

Section moved to the beginning.
Point 1 editted according to comment.
"Destination details" description added.
